auto_gptq/__init__.py

```diff
@@ -1,6 +1,4 @@
 from .modeling import AutoGPTQForCausalLM, BaseQuantizeConfig
 from .utils.exllama_utils import exllama_set_max_input_length
 from .utils.peft_utils import get_gptq_peft_model
-
-
-__version__ = "0.8.0.dev0"
+from .version import __version__
```

Comment:
@fxmarty  Can you check if there is a missing file `version` push on your end? This line and __base.py with same new version code is breaking pkg.

Reply:
yes it was likely missing my bad